textmining.py

```diff
@@ -0,0 +1,71 @@
+# most common words
+# this code computes the most common words of the adventure_of_sherlock_holmes
+import pickle
+import requests
+import string
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+adventure_of_sherlock_holmes = requests.get('http://www.gutenberg.org/ebooks/1661.txt.utf-8').text
+# Analyzing "Adventure of Sherlock Holmes"
+
+# Save data to a file
+f = open('adventure_of_sherlock_holmes.pickle', 'wb')
+pickle.dump(adventure_of_sherlock_holmes, f)
+f.close()
+
+# Load data from a file
+input_file = open('adventure_of_sherlock_holmes.pickle', 'rb')
+reloaded_copy_of_texts = pickle.load(input_file)
+print(type(reloaded_copy_of_texts))
+
+
+def process(s):
+    # creates a dictionary called hist by stripping of the text
+    hist = dict()
+    new_s = s.replace('-', ' ')
+    for word in new_s.split():
+        word = word.strip(string.punctuation + string.whitespace)
+        word = word.lower()
+        hist[word] = hist.get(word, 0) + 1
+    return hist
+
+
+hist = process(reloaded_copy_of_texts)
+print(hist)
+
+
+def most_common(text):
+    # creates the dictionary called t to find out most commonly used words
```

Comment:
This comment is misleading because it seems like what this function does is just sorting a dictionary by its value and returning the sorted dictionary as a list.